### PR TITLE
Updating Security tests

### PR DESCRIPTION
**Pull request by albertomisail**

### CODE REVIEW

File: jre_emul/test_sources.mk [hunk 1]
```diff
@@ -702,22 +713,39 @@ TEST_SOURCES := \
     org/apache/harmony/regex/tests/java/util/regex/PatternTest.java \
     org/apache/harmony/regex/tests/java/util/regex/ReplaceTest.java \
     org/apache/harmony/regex/tests/java/util/regex/SplitTest.java \
+	org/apache/harmony/security/tests/java/security/AlgorithmParameterGenerator2Test.java \
```
Comment (tomball) [inline jre_emul/test_sources.mk:716]: Fix indention.

File: jre_emul/test_sources.mk [hunk 2]
```diff
@@ -702,22 +713,39 @@ TEST_SOURCES := \
...
     org/apache/harmony/security/tests/java/security/AlgorithmParametersTest.java \
+    org/apache/harmony/security/tests/java/security/CodeSignerTest.java \
     org/apache/harmony/security/tests/java/security/DigestExceptionTest.java \
     org/apache/harmony/security/tests/java/security/DigestInputStream2Test.java \
     org/apache/harmony/security/tests/java/security/DigestInputStreamTest.java \
     org/apache/harmony/security/tests/java/security/DigestOutputStreamTest.java \
+    org/apache/harmony/security/tests/java/security/GeneralSecurityExceptionTest.java \
+    org/apache/harmony/security/tests/java/security/GuardedObjectTest.java \
+    org/apache/harmony/security/tests/java/security/InvalidAlgorithmParameterExceptionTest.java \
+	org/apache/harmony/security/tests/java/security/InvalidKeyExceptionTest.java \
```
Comment (tomball) [inline jre_emul/test_sources.mk:727]: Indention.

File: jre_emul/test_sources.mk [hunk 3]
```diff
@@ -702,22 +713,39 @@ TEST_SOURCES := \
...
+	org/apache/harmony/security/tests/java/security/KeyExceptionTest.java \
     org/apache/harmony/security/tests/java/security/KeyFactorySpiTest.java \
     org/apache/harmony/security/tests/java/security/KeyFactoryTest.java \
+    org/apache/harmony/security/tests/java/security/KeyManagementExceptionTest.java \
+    org/apache/harmony/security/tests/java/security/KeyPairGenerator1Test.java \
+    org/apache/harmony/security/tests/java/security/KeyPairGenerator2Test.java \
+    org/apache/harmony/security/tests/java/security/KeyPairGenerator3Test.java \
     org/apache/harmony/security/tests/java/security/KeyPairGeneratorSpiTest.java \
     org/apache/harmony/security/tests/java/security/KeyPairTest.java \
+	org/apache/harmony/security/tests/java/security/KeyStore4Test.java \
```
Comment (tomball) [inline jre_emul/test_sources.mk:738]: Indention.

File: jre_emul/test_sources.mk [hunk 4]
```diff
@@ -702,22 +713,39 @@ TEST_SOURCES := \
...
+	org/apache/harmony/security/tests/java/security/KeyStoreExceptionTest.java \
     org/apache/harmony/security/tests/java/security/KeyTest.java \
     org/apache/harmony/security/tests/java/security/MessageDigest1Test.java \
     org/apache/harmony/security/tests/java/security/MessageDigest2Test.java \
     org/apache/harmony/security/tests/java/security/MessageDigestSpiTest.java \
+    org/apache/harmony/security/tests/java/security/NoSuchAlgorithmExceptionTest.java \
+	org/apache/harmony/security/tests/java/security/NoSuchProviderExceptionTest.java \
```
Comment (tomball) [inline jre_emul/test_sources.mk:745]: Indention.

Comment (albertomisail) [reply jre_emul/test_sources.mk]: Done. Also rebased with upstream/master

**Pull request merged**